### PR TITLE
updated stylesheet to prevent horizontal scrolling (#2)

### PR DESCRIPTION
**Pull request by ulieckstein**

### CODE REVIEW

File: assets/css/home.css [hunk 1]
```diff
@@ -45,7 +50,7 @@ html {
 /* CUSTOM SCROLLBAR */
 
 ::-webkit-scrollbar {
-    width: 10px;
+    width: 0;
```
Comment (jeelpatel17) [inline assets/css/home.css]: You can't remove the scrollbar, its very important according to site usability!

**Comment by ulieckstein** [comment]
restored original scrollbar width, should work fine now

**Comment by ulieckstein** [comment]
ah, i see that #3 already solved the issue in the meantime. think this can be closed then

**Comment by jeelpatel17** [comment]
@ulieckstein Yes, but you gave me a new perspective on how it can be solved, thanks!

**Pull request closed without merging**